crates/bevy_input/Cargo.toml

```
@@ -20,4 +20,4 @@ bevy_math = { path = "../bevy_math", version = "0.8.0-dev" }
 bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
 
 # other
-serde = { version = "1", features = ["derive"], optional = true }
+serde = { version = "1", features = ["derive"], optional = true }
```

```suggestion
serde = { version = "1", features = ["derive"], optional = true }

```
This will get rid of the noise in the diff.